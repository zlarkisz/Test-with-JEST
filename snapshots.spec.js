const user = {
  name: "Jane Doee",
  age: 30,
  job: "Software Engineer",
};

test("user matches snapshot", () => {
  expect(user).toMatchSnapshot();
});
