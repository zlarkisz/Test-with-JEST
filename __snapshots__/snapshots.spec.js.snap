// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user matches snapshot 1`] = `
{
  "age": 30,
  "job": "Software Engineer",
  "name": "Jane Doee",
}
`;
